### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by slimbuck**
This is an initial PR updating most UI elements in Supersplat.

Almost all existing functionality is kept and a followup PR will complete the redesign started here before release.

In addition to the UI update, this PR makes a few other code changes:
* much of the styling is moved from the monolith style.scss and split into more modular files for easier maintenance
* sphere selector tool looks different
* package updated to v1 alpha
* rect and picker tool have been combined

Tools are moved onto a tool panel:
Screenshot 2024-08-05 at 16 23 36

Other options have moved to a new main menu:
Screenshot 2024-08-05 at 16 23 52

Scene manager panel controls splats and their transform:
Screenshot 2024-08-05 at 16 24 04

The view panel groups together view-related things.
Screenshot 2024-08-05 at 16 24 13

The remaining bits needed to finish off this work:
* update svg icons
* add splash screen / help dialog
* styling of data panel
* fix menu and other styling not working in safari, including shadows and menu single-click

### CODE REVIEW

File: src/tools/sphere-selection.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            if (canvas) {
+                const w = canvas.clientWidth;
+                const h = canvas.clientHeight;
+                gizmo.size = 1200 / Math.max(w, h);
+
+                // FIXME:
+                // this is a temporary workaround to undo gizmo's own auto scaling.
+                // once gizmo's autoscaling code is removed, this line can go too.
+                // @ts-ignore
+                gizmo._deviceStartSize = Math.min(scene.app.graphicsDevice.width, scene.app.graphicsDevice.height);
```
Comment (willeastcott) [inline src/tools/sphere-selection.ts:78]: Is this workaround still needed? cc @kpal81xd

File: src/ui/splat-list.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-import { Container, Label, Element } from 'pcui';
+import { Container, Label, Element as PcuiElement } from 'pcui';
+import { Events } from '../events';
+import { Splat } from '../splat';
+import { Element, ElementType } from '../element';
```
Comment (willeastcott) [inline src/ui/splat-list.ts:4]: Any chance of renaming the app's `Element` class since it clashes with PCUI?

**Review by willeastcott — APPROVED** [review]
Nice! 🔥

File: src/ui/splat-list.ts (hunk 2 above)
Comment (slimbuck) [reply src/ui/splat-list.ts:4]: Could do. What would you recommend? Perhaps "Entity"? :)

Comment (willeastcott) [reply src/ui/splat-list.ts:4]: LOL. That's taken too!

Comment (LeXXik) [reply src/ui/splat-list.ts:4]: Section, piece, segment, bit, feature, detail, unit, part

File: src/tools/sphere-selection.ts (hunk 1 above)
Comment (kpal81xd) [reply src/tools/sphere-selection.ts:86]: Yea this value no longer exists. The custom scaling is does based on the `size` prop

File: src/ui/splat-list.ts (hunk 2 above)
Comment (kpal81xd) [reply src/ui/splat-list.ts:4]: NativeElement

**Pull request merged**